service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -239,42 +205,92 @@ void shouldAttemptGrantAccessToAllLocalAuthorityUsersWhenGrantAccessFailsForSome
     }
 
     @Test
-    void shouldSetPageShowToYesWhenToggleIsEnabled() {
-        given(featureToggleService.isMigrateToManageOrgWarningPageEnabled(anyString())).willReturn(true);
+    void shouldNotValidateWhenToggleIsDisabled() {
+        given(featureToggleService.isAllowCaseCreationForUsersNotOnboardedToMOEnabled(anyString())).willReturn(false);
 
         CaseDetails caseDetails = CaseDetails.builder()
             .data(Map.of("caseName", "title"))
             .build();
 
-        AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToStartEvent(caseDetails);
+        AboutToStartOrSubmitCallbackResponse response = postMidEvent(caseDetails);
```

Comment:
our convention is that different callbacks are in separate files. If you want to keep it in single file thats fine, but method name should tell maybe that is is about mid event

Reply:
have moved